diff --git a/node_modules/@types/consul/index.d.ts b/node_modules/@types/consul/index.d.ts
index 18355be..831d45b 100644
--- a/node_modules/@types/consul/index.d.ts
+++ b/node_modules/@types/consul/index.d.ts
@@ -252,6 +252,7 @@ declare namespace Consul {
                 ttl?: string;
                 notes?: string;
                 status?: string;
+                tcp?: string;
             }
 
             interface ListOptions extends CommonOptions { }
@@ -719,6 +720,11 @@ declare namespace Consul {
         }
 
         interface DeleteOptions extends DelOptions { }
+
+        interface GetOneResponse {
+            Value: string,
+            ModifyIndex: string,
+        }
     }
 
     interface Kv {
@@ -956,6 +962,7 @@ declare namespace Consul {
         
         interface WatchOptions {
             key?: string;
+            recurse?: boolean;
         }
 
         interface Options {
