diff --git a/node_modules/@types/consul/index.d.ts b/node_modules/@types/consul/index.d.ts
index 18355be..d5f3b48 100644
--- a/node_modules/@types/consul/index.d.ts
+++ b/node_modules/@types/consul/index.d.ts
@@ -719,6 +719,11 @@ declare namespace Consul {
         }
 
         interface DeleteOptions extends DelOptions { }
+
+        interface GetOneResponse {
+            Value: string,
+            ModifyIndex: string,
+        }
     }
 
     interface Kv {
@@ -956,6 +961,7 @@ declare namespace Consul {
         
         interface WatchOptions {
             key?: string;
+            recurse?: boolean;
         }
 
         interface Options {
